src/courseware/course/sequence/Sequence.jsx

```diff
@@ -77,6 +78,20 @@ function Sequence({
     unitNavigationHandler(destinationUnitId);
   };
 
+  function handleSidebarNavigation(event) {
+    if (event.data.message === 'outline_sidebar_navigation_started') {
+      sidebarNavigationClickHandler(event.data.subsection_id);
+    }
+  }
+  // Event Listener for frontend-app-sidebar-navigation
+  useEffect(() => {
+    window.addEventListener('message', handleSidebarNavigation);
+    // Cleanup eventListener
+    return () => {
+      window.removeEventListener('message', handleSidebarNavigation);
```

Comment:
Why do you need to call handleSidebarNavigation" again? @anfbermudezme

Reply:
Is the correct way to do it according to this documentation https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup

Reply:
Nice! Thanks. @anfbermudezme